devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy

```diff
@@ -21,9 +21,11 @@ metadata {
 		capability "Sensor"
 		capability "Battery"
 		capability "Health Check"
+		//capability "Tamper Alert" waiting for LE fix for Zooz 4-in-1
```

Comment:
Sorry I didn't realize. This isn't the right approach. 

We should just give the Zooz fingerprint-specific metadata pointing to "generic-motion-6", which includes everything except Tamper